blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/RollupRunnableIntegrationTest.java

```diff
@@ -95,7 +97,11 @@ public void setUp() throws Exception {
             BluefloodSetRollup rollup = new BluefloodSetRollup().withObject(i);
             metric = new PreaggregatedMetric(time, setLocator, ttl, rollup);
             preaggregatedMetrics.add(metric);
-            
+
```

Comment:
I just wanted to generate a long count for the values, so I just multiplied i+1 by 5 and cast it to long.